public/static/locales/en/common.json

```diff
@@ -145,5 +145,139 @@
+  "minimum_booking_notice": "Minimum booking notice",
+  "delete_event_type_description": "Are you sure you want to delete this event type? Anyone who you&apos;ve shared this link with will no longer be able to book using it.",
+  "delete_event_type": "Delete Event Type",
+  "confirm_delete_event_type": "Yes, delete event type",
+  "connect_new_app": "Connect a new App",
+  "integrate_your_account_services": "Integrate your account with other services.",
+  "add": "Add",
+  "select_calendars": "Select Calendars",
+  "if_no_entry_all_calendars_checked": "If no entry is selected, all calendars will be checked",
+  "connect_caldav_server": "Connect to CalDav Server",
+  "your_credentials_stored_encrypted": "Your credentials will be stored and encrypted.",
+  "connect_apple_server": "Connect to Apple Server",
+  "generate_app_spec_password": "Generate an app specific password to use with cal.com at",
+  "integrations": "Integrations",
+  "connect_favourite_apps": "Connect your favourite apps.",
+  "calendar_integration": "Calendar Integration",
+  "video_conferencing": "Video Conferencing",
+  "connected": "Connected",
+  "not_connected": "Not Connected",
+  "dont_have_apps_connected": "You don&apos;t have any apps connected.",
+  "dont_have_apps_connected_description": "You currently do not have any apps connected. Connect your first app to get started.",
+  "select_which_calendars_checked_double_booking": "Select which calendars are checked for availability to prevent double bookings.",
+  "launch_own_app": "Launch your own App",
+  "if_want_add_your_app_touch_us": "If you want to add your own App here, get in touch with us.",
+  "contact_us": "Contact us",
+  "integrations_details": "Integrations Details",
+  "manage_delete_this_app": "Manage and delete this app.",
+  "name_app_prop": "{`${getIntegrationName(props.integration.type)} App`}",
+  "name_app_description": "Information about your {getIntegrationName(props.integration.type)} App.",
+  "app_name": "App Name",
+  "app_category": "App Category",
+  "delete_this_app": "Delete this App",
+  "delete_this_app_description": "Once you delete this app, it will be permanently removed.",
+  "delete_app": "Delete App",
+  "end_time": "End Time",
+  "start_time": "Start Time",
+  "start_and_end_times_successfully": "The start and end times for your day have been changed successfully.",
+  "set_start_and_end_time_minimum_meeting": "Set the start and end time of your day and a minimum buffer between your meetings.",
+  "change_available_times": "Change your available times",
+  "launch_troubleshooter": "Launch troubleshooter",
+  "troubleshooter_availability_explore_times": "Troubleshoot your availability to explore why your times are showing as they are.",
+  "something_does_look_right": "Something doesn&apos;t look right?",
+  "currently_start_end_time": "Currently, your day is set to start at {convertMinsToHrsMins(user.startTime)} and end at{" "} {convertMinsToHrsMins(user.endTime)}.",
```

Comment:
This will never work. You need to pass in dynamic variables as
```
"Currently, your day is set to start at {{startAt}} and end at {{endAt}}."
```
and then in the component you would use it as 
```
{t('currently_start_end_time', { startAt: convertMinsToHrsMins(user.startTime), endAt: convertMinsToHrsMins(user.endTime))}
```